Formula/s/spirv-headers.rb

```diff
@@ -29,6 +29,8 @@ def install
     cp pkgshare/"tests/example.cpp", testpath
 
     (testpath/"CMakeLists.txt").write <<~EOS
+      cmake_minimum_required(VERSION #{Formula["cmake"].version})
```

Comment:
Could probably just hardcode a version here, TBH.

Reply:
yeah, that is true. I was debating a bit.